mobly/controllers/android_device.py

```diff
@@ -567,14 +558,14 @@ def load_snippet(self, name, package):
         self._snippet_clients[name] = client
         setattr(self, name, client)
 
-    def _start_sl4a(self):
-        """Create an sl4a connection to the device.
+    def load_sl4a(self):
```

Comment:
The examples in readme.md will need updating accordingly since they use sl4a currently.